### DISCUSSION TITLE
where is the actual activity recorded located at?Is it modular*?

### DISCUSSION THREAD

**Discussion by RESDXChgfore9hing** [ActivityWatch/activitywatch#795] [Q&A] [answered]
windows i think its at %APPDATA% but which folder or file?is it the db file?

**Comment by BelKed** [comment, marked as answer]
Recorded activity is saved in an SQLite database.
You can find the database file in `{AW_FOLDER}/aw-server/peewee-sqlite.v2.db`

Reference: https://docs.activitywatch.net/en/latest/directories.html#data
